app/views/overview/_service-instance-row.html

```diff
@@ -1,48 +1,45 @@
-<div class="list-pf-item" ng-class="{ active: row.expanded }">
+<div class="list-pf-item provisioned-service" ng-class="{ active: row.expanded }">
   <div class="list-pf-container" ng-click="row.toggleExpand($event)">
-
     <div class="list-pf-chevron">
       <div ng-include src=" 'views/overview/_list-row-chevron.html' " class="list-pf-content"></div>
     </div>
-
     <div class="list-pf-content">
       <div class="list-pf-name">
         <h3>
           <span ng-bind-html="row.displayName | highlightKeywords : row.state.filterKeywords"></span>
-          <div class="component-label tranform-none">
+          <div class="list-row-longname">
             {{row.apiObject.metadata.name}}
           </div>
         </h3>
         <div class="status-icons">
           <notification-icon ng-if="!row.expanded" alerts="row.notifications"></notification-icon>
         </div>
       </div>
-    </div>
-
-    <div class="list-pf-content hidden-xs hidden-sm">
-      <div class="list-pf-content-right">
-        <div>
-          <strong ng-if="!row.bindings.length">No Bindings</strong>
-          <strong ng-if="row.bindings.length">Bindings</strong>
+      <div class="list-pf-details">
+        <div ng-if="!row.expanded">
+          <div class="hidden-xs hidden-sm">
+            <span ng-if="!row.bindings.length">
+              <a href="" ng-click="row.showOverlayPanel('bindService', {target: row.apiObject})">Create Binding</a>
+            </span>
+            <span ng-if="row.bindings.length" class="component-label">Bindings</span>
+            <p ng-if="row.bindings.length >= 1" class="bindings">
+              {{row.bindings[0].metadata.name}}<span ng-if="row.bindings.length > 1">, </span>
+              <a
+                ng-if="row.bindings.length > 1"
+                ng-click="row.toggleExpand($event, true)">
+                {{row.bindings.length -1}} other<span ng-if="row.bindings.length > 2">s</span></a>
+            </p>
+          </div>
+        </div>
+        <div
+          class="hidden-xs"
```

Comment:
Not new in this PR, but we should give mobile users a way to get to the dashboard for the service.